<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="BGLabelClass" name="MainLogo">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/main/logo.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="BGLabelClass" name="bgImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/main/bg.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="MadeBy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>110</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Made by&lt;br/&gt;Alrott SlimRG&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="VersionLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Ver. 2.0.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="CloseLabelClass" name="CloseLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/dynamic/close/main.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="SettingsLabelClass" name="SettingsLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/dynamic/settings/main.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="SettingsFrameClass" name="SettingsFrame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>441</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="SettingsLogoLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>7</y>
       <width>441</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:400; color:#7027d8;&quot;&gt;SETTINGS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Settings_ProxyLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#dfb211;&quot;&gt;Proxy:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ProxyIP">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 0, 255);
border-color: rgb(170, 85, 255);</string>
     </property>
     <property name="text">
      <string>192.168.1.1</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ProxyPort">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 0, 255);
border-color: rgb(170, 85, 255);</string>
     </property>
     <property name="text">
      <string>8080</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Settings_ProxyLabel_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#e6e6e6;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Settings_ProxyType">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>190</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 0, 255);
border-color: rgb(170, 0, 255);</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>SOCKS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HTTP(S)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="Settings_ProxyLabel_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#e6e6e6;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Settings_TranslationLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#dfb211;&quot;&gt;Language online transation:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Settings_UseTranslation">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Settings_SaveButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 0, 255);</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="Settings_UseProxyLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#dfb211;&quot;&gt;Use proxy:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Settings_UseProxy">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Settings_CheckProxy">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 0, 255);</string>
     </property>
     <property name="text">
      <string>Check proxy</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="LoadingBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="LoadingLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>250</width>
       <height>250</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(115, 0, 172);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="LoadingAnimationLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>49</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;Определение языка:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="WarningMsg">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>271</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 20, 20);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="WarningLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>231</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;WARNING!!!&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;You have no internet conncection.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Programm will run in Offline mode.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;You can change network parameters in settings page.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="CloseMsgButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 0, 255);
</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>bgImage</zorder>
   <zorder>MainLogo</zorder>
   <zorder>MadeBy</zorder>
   <zorder>VersionLabel</zorder>
   <zorder>CloseLabel</zorder>
   <zorder>SettingsLabel</zorder>
   <zorder>SettingsFrame</zorder>
   <zorder>LoadingBox</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CloseLabelClass</class>
   <extends>QLabel</extends>
   <header location="global">closelabelclass.hpp</header>
  </customwidget>
  <customwidget>
   <class>BGLabelClass</class>
   <extends>QLabel</extends>
   <header location="global">bglabelclass.hpp</header>
  </customwidget>
  <customwidget>
   <class>SettingsLabelClass</class>
   <extends>QLabel</extends>
   <header location="global">settingslabelclass.hpp</header>
  </customwidget>
  <customwidget>
   <class>SettingsFrameClass</class>
   <extends>QFrame</extends>
   <header location="global">settingsframeclass.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
